test/export/test_export.py

```diff
@@ -7309,6 +7312,95 @@ def forward(self, x):
         with self.assertRaisesRegex(error, error_msg):
             _ = export(f, (torch.tensor(6),))
 
+    def test_is_non_negative_check_function(self):
+        import sympy as sp
+
+        from torch.fx.experimental.symbolic_shapes import _is_non_negative_check
+
+        x = sp.Symbol("x")
+        variable_name = sp.Symbol("variable_name")
+        obj_attr = sp.Symbol("obj.attr")
+        tensor_shape = sp.Symbol("tensor.shape[0]")
+
+        self.assertEqual(_is_non_negative_check(x >= 0), "x")
```

Comment:
I am unsure whats the differece between the tests using tensor_shape, and obj_attr since they are all sympy symbols now and we do not text based matching. 
before you merge can you ensure removing not needed ones.